README.md

```diff
@@ -118,6 +97,12 @@ final_output: Dict containing either:<br>
             {"success": True, "data": < stage 2 output - json >}<br>
             {"success": False, "error": < error message - string >}<br>
 
+### Stage 3
+TODO
+
+### Stage 4
+TODO
+
```

Comment:
Maybe say "N/A - Currently mocked out" instead, as "TODO" makes it seem like you wanted to implement it but forgot.

Reply:
Updated.